db/schema.rb

```diff
@@ -11,19 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160320203038) do
+ActiveRecord::Schema.define(version: 20160417233633) do
```

Comment:
We may want to remove schema.rb from the repository if its going to be incapable of defining the schema when there are custom types.